### ISSUE TITLE
On Date page, “1 of 6” is not labelled as a header. Blind participants don’t know how many steps are next.

### ISSUE THREAD

**Issue by melbanyard** [cds-snc/report-a-cybercrime#950]
_"steps 1 of 6... it would be handy if this was a heading"_

**Comment by amazingphilippe** [comment]
I'll leave this here for reference : [Multi-page Forms](https://www.w3.org/WAI/tutorials/forms/multi-page/)
Gives different possible solutions for this issue

**Comment by amazingphilippe** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Apply findings from Nov Usability Testing w/ Blind and low vision participants (to existing prototype)](https://trello.com/c/EDa35Fdi/51-apply-findings-from-nov-usability-testing-w-blind-and-low-vision-participants-to-existing-prototype)

**Comment by DianeLiu2019** [comment]
this task is competed , associate with PR #984
